### DISCUSSION TITLE
咸鱼花 | /cn/2022/04/fish-flower/

### DISCUSSION THREAD

**Discussion by giscus** [rbind/yihui#1509] [Comments]
我在红楼中做了个梦 听湘云喊我
去翻一条咸鱼
本瞎厨迷惑地躺完一枪
摸索着穿过大观园
凭着嗅觉
铲翻这条咸鱼
不料
这鱼翻身惊坐起
祭出一圈锅贴
又喊人帮啃这
八个锅贴
要沿着边
阿木阿木阿木
依次啃缺三口
两眼一抹黑的瞎厨
再次凭多年的嗅觉
啃完锅贴
不料
这锅里
咸鱼化身花蕊
锅贴变成一圈花瓣
原来
当脑洞开得精致时
咸鱼也会开花
最后
我从一朵
咸鱼烙饼花中醒来

https://yihui.org/cn/2022/04/fish-flower/

**Comment by JinjiPang** [comment]
我感觉这个就是活鱼锅贴的原身。

**Reply by yihui** [reply]
是的，注释中有活鱼锅贴。后来我做[烤鱼](https://yihui.org/cn/2022/05/grill-fish/)的时候也确实贴过玉米面饼。
